readme.md

```diff
@@ -207,6 +207,22 @@ Note that it's `@Default`, not `@Defaults`.
 
 You cannot use `@Default` in an `ObservableObject`. It's meant to be used in a `View`.
 
+#### `@Default` in `@Observable`
+
+With the `@Default` macro, you can use `Defaults` inside `@Observable` classes that use the [Observation](https://developer.apple.com/documentation/observation) framework. Doing so is as simple as importing `DefaultsMacros` as adding two lines to a property (note that adding `@ObservationIgnored` is needed to prevent clashes with `@Observable`):
+
+```swift
+import Defaults
+import DefailtsMacros
```

Comment:
Minor typo in the example, which should read `import DefaultsMacros`.

Thanks for the PR.  I'm new to SwiftUI and just tried this out.